deepchem/models/torch_models/layers.py

```diff
@@ -3056,3 +3056,87 @@ def forward(self, inputs: torch.Tensor):
         atom_enbeddings = torch.nn.functional.embedding(atom_number,
                                                         self.embedding_list)
         return atom_enbeddings
+
```

Comment:
The new layer needs to be added to the docs. Check the other layer PRs to see how they do it